class_generator/class_generator.py

```diff
@@ -281,13 +271,13 @@ def generate_resource_file_from_dict(
         if not os.path.exists(tests_path):
             os.makedirs(tests_path)
 
-        _output_file = os.path.join(tests_path, f"{formatted_kind_str}_res.py")
+        _output_file = os.path.join(tests_path, f"{output_file_prefix}{formatted_kind_str}_res.py")
 
     elif output_file:
         _output_file = output_file
 
     else:
-        _output_file = os.path.join("ocp_resources", f"{formatted_kind_str}.py")
+        _output_file = os.path.join("ocp_resources", f"{output_file_prefix}{formatted_kind_str}.py")
```

Comment:
```suggestion
        _output_file = os.path.join("ocp_resources", f"{output_file_prefix}_{formatted_kind_str}.py")
```
?

Reply:
shouldn't the filename start with the resource name and not the group, so when looking at the list of files, they are next to each other?